gateways/clients/responses/hathor_wallet_service/hathor_wallet_service_token_response.py

```diff
@@ -0,0 +1,161 @@
+from dataclasses import dataclass
+from typing import List
+
+from dacite import from_dict
+
+from domain.tx.token import Token, TokenList, TokenListItem, TokenTransactionList, TokenTransactionListItem
+
+
+@dataclass
+class HathorWalletServiceTokenResponse:
+    """Response from hathor wallet service when requesting a token
+
+    :param id: id of token
+    :type id: str
```

Comment:
Are we generating a docs with these comments? If we are, is it possible to make it use the type annotations instead of having to duplicate the information with `:type`? If we aren't I feel like we have no feedback if we're making a mistake or not, and I wouldn't bother with `:type` at all, but the descriptions are nice.